docs/style-guides/js.md

```diff
@@ -0,0 +1,252 @@
+# origin-web-console JS style guide
+
+A reasonable approach to assisting in style consistency in `openshift-web-console`
+inspired by guides such as the style guide published by [airbnb](https://github.com/airbnb/javascript).
+
+## ESLint
+
+Run `grunt eslint` to check code for errors and style discrepancies.
+
+## Examples
+
+
+### Variables
+
+Group your variable names but use individual `var` statements:
+
+```javascript
+
+// bad
+var foo = function() {
+  var bar = 1;
+  if(bar === baz) {
+    doStuff();
+  }
+  var qux = 2;
+  // more code...
+  var quux = 4;
+  // more code...
+}
+
+// good
+var foo = function() {
+  var bar = 1;
+  var qux = 2;
+  var quux = 4;
+
+  if(bar === baz) {
+    doStuff();
+  }
+  // more code...
+}
+
+```
+
+
+### Comparison Operators
+
+Use `===` and `!==`.  Avoid `==` and `!=`.  This will avoid type coercion bugs:
+
+```javascript
+
+// bad
+if(foo == bar) {
```

Comment:
I personally prefer spaces after the `if`, and it's what most of the code uses today.